hmpps-prisoner-search/build.gradle.kts

```diff
@@ -77,3 +77,35 @@ tasks {
     }
   }
 }
+
+tasks.register<Test>("attributeSearchIntegrationTest") {
+  // Inherit configurations from the existing 'test' task
+  val testTask = tasks.named("test").get() as Test
+  testClassesDirs = testTask.testClassesDirs
+  classpath = testTask.classpath
+
+  useJUnitPlatform()
+
+  // Override test class include patterns if necessary
+  includes.clear()
+  include("**/AttributeSearchIntegrationTest.*")
+
+  // Force the task to always run
+  outputs.upToDateWhen { false }
+}
+
+tasks.register<Test>("allOtherTests") {
+  // Inherit configurations from the existing 'test' task
+  val testTask = tasks.named("test").get() as Test
+  testClassesDirs = testTask.testClassesDirs
+  classpath = testTask.classpath
+
+  useJUnitPlatform()
+
+  // Override test class exclude patterns if necessary
+  excludes.clear()
+  exclude("**/AttributeSearchIntegrationTest.*")
+
+  // Force the task to always run
+  outputs.upToDateWhen { false }
+}
```

Comment:
2 new Gradle tasjks - one to `include("**/AttributeSearchIntegrationTest.*")` and one to `exclude("**/AttributeSearchIntegrationTest.*")`.

This means I don't have to mess with the `test` task - it still runs all the tests locally if that's what you want to do.